website/static/js/pages/render-nodes.js

```diff
@@ -2,9 +2,12 @@
 
 var $osf = require('js/osfHelpers');
 var $ = require('jquery');
+var ko = require('knockout');
 
 // model for components, due to simplicity did not create a new file
-var ComponentControl = {};
+var ComponentControl = {
+     anonymousUserName:ko.observable('<em>A user</em>')
```

Comment:
I don't think a variable will be necessary here at all - see my [other comment](https://github.com/CenterForOpenScience/osf.io/pull/3234/files#r32860935).